src/test/java/org/cryptomator/cryptofs/DeleteNonEmptyCiphertextDirectoryIntegrationTest.java

```diff
@@ -159,21 +161,35 @@ private Path firstEmptyCiphertextDirectory() throws IOException {
 		try (Stream<Path> allFilesInVaultDir = Files.walk(pathToVault)) {
 			return allFilesInVaultDir //
 					.filter(Files::isDirectory) //
-					.filter(this::isEmptyDirectory) //
+					.filter(this::isEmptyCryptoFsDirectory) //
 					.filter(this::isEncryptedDirectory) //
 					.findFirst() //
 					.get();
 		}
 	}
 
-	private boolean isEmptyDirectory(Path path) {
+	private boolean isEmptyCryptoFsDirectory(Path path) {
+		Predicate<Path> isIgnoredFile = p -> Constants.DIR_ID_FILE.equals(p.getFileName().toString());
 		try (Stream<Path> files = Files.list(path)) {
-			return files.filter(p -> p.getFileName().toString().equals(Constants.DIR_ID_FILE)).count() == 0;
+			return files.noneMatch(isIgnoredFile.negate());
 		} catch (IOException e) {
 			throw new UncheckedIOException(e);
 		}
 	}
 
+	@Test
+	@DisplayName("Tests internal cryptofs directory emptiness definition")
+	public void testCryptoFsDirEmptiness() throws IOException {
+		var emptiness = pathToVault.getParent().resolve("emptiness");
+		var ignoredFile = emptiness.resolve(Constants.DIR_ID_FILE);
+		Files.createDirectory(emptiness);
+		Files.createFile(ignoredFile);
+
+		boolean result = isEmptyCryptoFsDirectory(emptiness);
+
+		Assertions.assertTrue(result, "Directory containing only dir id file is not considered empty");
```

Comment:
```suggestion
		Assertions.assertTrue(result, "Directory containing only dir id file is considered empty");
```

Reply:
This is correct, although not best english:

The message is shown in failure case.
It is a failure, if a directory only containing the dir id file is not marked/shown/indicated/considered empty.

Reply:
Oh now I understand. 💡 This is in fact misleading. However, in most cases we just do the assertions without any comment at all, since the `@DisplayName` of the test case should be sufficient.